stories/module-adsense-settings.stories.js

```diff
@@ -98,6 +98,7 @@ storiesOf( 'AdSense Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```

Comment:
Same with settings. We need padding for settings to make it look good in the storybook. Please, update all module stories to have the default padding for their settings stories.